src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -1647,63 +1648,3 @@ int TriangleCountExecutor::getUid() {
     static std::atomic<std::uint32_t> uid{0};
     return ++uid;
 }
-
-int TriangleCountExecutor::collectPerformaceData(PerformanceSQLiteDBInterface *perDB, std::string graphId,
```

Comment:
Why do we delete this function?

Reply:
PageRankExecutor also needs this function. Therefore,I added this on it's parent class AbstractExecutor

Reply:
Ok